01/app.js

```diff
@@ -1,11 +1,28 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Table, TableHeader, TableBody, TableRow, TableFooter } from './Components/Table';
 
-import Table from './Table';
+import { v4 as uuidv4 } from 'uuid';
 import data from './data.json';
 
-function App()  {
-    return <Table data={ data }/>
+const App = () => {
+    return (
+        <Table>
+            <TableHeader names={['ID', 'Name', 'Price', 'Quantity', 'Total Price:']} />
+            <TableBody>
+                {data.map(row =>
+                    <TableRow
+                        key={uuidv4()}
```

Comment:
Zdecydowanie lepiej użyć tutaj `row.id`.
`uuid` generujemy jak dodajemy nowy element i to `uuid` trzymamy do tego elementu zawsze takie same (zazwyczaj w `state`)